db/schema.rb

```diff
@@ -67,17 +74,7 @@
     t.index ["name"], name: "index_groups_on_name", unique: true
   end
 
-  create_table "help_texts", force: :cascade do |t|
```

Comment:
should this have been dropped?

Reply:
I think that it was already dropped but left around lingering in this file

Reply:
```
git grep help_text -- db/migrate
```

only shows column names for this, not a separate table